docs/design/sql/08-expressions.md

```diff
@@ -0,0 +1,192 @@
+# SQL Expressions
+
+## Overview
+
+Expressions is one of the core elements of every SQL implementation.
```

Comment:
Expressions _**are**_

Reply:
fixed in d1d2929413835f7a462bc88686630dd57203b27c